src/views/ipv/page/no-photo-id-abandon-find-another-way.njk

```diff
@@ -0,0 +1,101 @@
+{% extends "shared/base.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+
+{% set pageTitleKey = 'pages.noPhotoIdAbandonFindAnotherWay.title' %}
+{% set googleTagManagerPageId = "noPhotoIdAbandonFindAnotherWay" %}
+
+{% set errorState = pageErrorState %}
+{% set errorTitle = 'pages.noPhotoIdAbandonFindAnotherWay.content.formErrorMessage.errorSummaryTitleText' | translate %}
+{% set errorText = 'pages.noPhotoIdAbandonFindAnotherWay.content.formErrorMessage.errorSummaryDescriptionText' | translate %}
+{% set errorHref = "#noPhotoIdAbandonFindAnotherWayForm" %}
+
+{% set photoIdHtml %}
+  {% include "components/types-of-photo-id.njk" %}
+{% endset %}
+
+{% block content %}
+  <h1 class="govuk-heading-l" id="header" data-page="{{ googleTagManagerPageId }}">{{ 'pages.noPhotoIdAbandonFindAnotherWay.header' | translate }}</h1>
+
+  <h2 class="govuk-heading-m">{{ 'pages.noPhotoIdAbandonFindAnotherWay.content.subHeading' | translate }}</h2>
+  <p class="govuk-body">{{ 'pages.noPhotoIdAbandonFindAnotherWay.content.paragraph1' | translate }}</p>
+
+  <h2 class="govuk-heading-m">{{ 'pages.noPhotoIdAbandonFindAnotherWay.content.subHeading2' | translate }}</h2>
+  <p class="govuk-body">{{ 'pages.noPhotoIdAbandonFindAnotherWay.content.paragraph2' | translate }}</p>
+
+  <ul class="govuk-list govuk-list--bullet">
+    {% for listItem in 'pages.noPhotoIdAbandonFindAnotherWay.content.list' | translate({ returnObjects: true }) %}
+      <li>{{ listItem }}</li>
+    {% endfor %}
+  </ul>
+
+  <p class="govuk-body">{{ 'pages.noPhotoIdAbandonFindAnotherWay.content.paragraph3' | translate }}</p>
+
+  {{ govukDetails({
+    summaryText: 'pages.noPhotoIdAbandonFindAnotherWay.content.details.label' | translate,
+    html: photoIdHtml,
+    id: 'typesOfPhotoIdDetails'
+  }) }}
+
+  <form id="noPhotoIdAbandonFindAnotherWay" action="/ipv/page/{{ pageId }}" method="POST">
+    <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+      {% set radioOptions = [
+        {
+          value: "app",
```

Comment:
Would a more explicit `mobileApp` fit? I think it may be more consistent with how we refer to it in other places

Reply:
Sure, makes sense